src/core/annotation.js

```diff
@@ -3326,6 +3484,28 @@ class FreeTextAnnotation extends MarkupAnnotation {
     super(parameters);
 
     this.data.annotationType = AnnotationType.FREETEXT;
+    this.setDefaultAppearance(parameters);
+    if (
+      !this.appearance &&
+      ((parameters.pdfManager &&
+        parameters.pdfManager.evaluatorOptions.isOffscreenCanvasSupported) ||
```

Comment:
Should we perhaps just add the following
```javascript
isOffscreenCanvasSupported:
  pdfManager.evaluatorOptions.isOffscreenCanvasSupported
```
to this structure https://github.com/mozilla/pdf.js/blob/7e5008f0ff013cc2b8f2af10db7f007e7cafe0de/src/core/annotation.js#L132-L146
so that we can remove these two lines here?